plugins/rbac/src/components/ConditionalAccess/ConditionsFormRow.tsx

```diff
@@ -2,17 +2,36 @@ import React from 'react';
 
 import { PermissionCondition } from '@backstage/plugin-permission-common';
 
-import { IconButton, makeStyles, useTheme } from '@material-ui/core';
+import {
+  FormControlLabel,
+  IconButton,
+  makeStyles,
+  Radio,
+  RadioGroup,
+  useTheme,
+} from '@material-ui/core';
 import AddIcon from '@mui/icons-material/Add';
 import RemoveIcon from '@mui/icons-material/Remove';
 import Box from '@mui/material/Box';
 import Button from '@mui/material/Button';
-import ToggleButton from '@mui/material/ToggleButton';
 import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
 
+import {
+  extractNestedConditions,
+  getDefaultRule,
+  nestedConditionButtons,
+  ruleOptionDisabled,
+} from '../../utils/conditional-access-utils';
+import { AddNestedConditionButton } from './AddNestedConditionButton';
 import { ConditionsFormRowFields } from './ConditionsFormRowFields';
 import { conditionButtons, criterias } from './const';
-import { ConditionsData, RuleParamsErrors, RulesData } from './types';
+import { CriteriaToggleButton } from './CriteriaToggleButton';
+import {
+  Condition,
+  ConditionFormRowProps,
+  ConditionsData,
+  NotConditionType,
+} from './types';
 
 const useStyles = makeStyles(theme => ({
```

Comment:
@ciiay since the styles have increased we can also think about moving it to a separate file.